src/main/java/com/bastagruppen/springskolsystem/model/Course.java

```diff
@@ -51,7 +56,7 @@ public static Course create(
         final Integer maxStudents,
         final List<Student> enrolledStudents
```

Comment:
This apparently gives an exception now. Need to annotate with @OneToMany or @ManyToOne and @JsonIgnore